pkgs/development/python-modules/scikit-learn/default.nix

```
@@ -66,10 +66,22 @@ buildPythonPackage rec {
 
   doCheck = !stdenv.isAarch64;
 
-  # Skip test_feature_importance_regression - does web fetch
-  disabledTests = [ "test_feature_importance_regression" ];
+  disabledTests = [
+    # Skip test_feature_importance_regression - does web fetch
+    "test_feature_importance_regression"
+    # NuSVC memmap tests causes segmentation faults in certain environments
+    # (e.g. Hydra Darwin machines) related to a long-standing joblib issue
+    # (https://github.com/joblib/joblib/issues/563). See also:
+    # https://github.com/scikit-learn/scikit-learn/issues/17582
+    "not (NuSVC and memmap)"
```

Just a wild guess, but don't we have to remove the `not` when using `disabledTests`?
```suggestion
    "(NuSVC and memmap)"
```